openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -824,6 +826,8 @@ The JWT MUST contain the following elements:
 
 The Credential Issuer MUST validate that the JWT used as a proof is actually signed by a key identified in the JOSE Header.
 
+If an `attestation` is provided and successfully validated by the Credential Issuer, it SHOULD return a Credential for each of the keys provided in the `attested_keys` claim of the attestation.
```

Comment:
Why only SHOULD?

Reply:
I believe the Credential Issuer always has the freedom to issue less credentials than were requested by the Wallet. I may not want Issuer to return 1000 Credentials if he requests so, or?

Reply:
Editors Call:
- there is max_batch_size in the Issuer's metadata
- general opinion that SHOULD is fine